app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people_initial: list) -> list:
+    people_list = [
+        Person(person["name"], person["age"])
+        for person in people_initial
+    ]
+    for person in people_initial:
+        if "wife" in person and person["wife"] is not None:
+            Person.people[person["name"]].wife = Person.people[person["wife"]]
```

Comment:
```suggestion
        if wife_name := person.get("wife"):
            Person.people[person["name"]].wife = Person.people[wife_name]
```
Cool approach but it is not necessary